website/src/documentation/index.md

```diff
@@ -29,7 +29,7 @@ This section provides in-depth conceptual information and reference material for
 
 Learn about the Beam Programming Model and the concepts common to all Beam SDKs and Runners.
 
-* The [Programming Guide]({{ site.baseurl }}/documentation/programming-guide/) introduces all the key Beam concepts.
+* Read through the [Programming Guide]({{ site.baseurl }}/documentation/programming-guide/) to get introduced to all the key Beam concepts.
 * Learn about Beam's [execution model]({{ site.baseurl }}/documentation/execution-model/) to better understand how pipelines execute.
 * Visit [Learning Resources]({{ site.baseurl }}/documentation/resources/learning-resources) for some of our favorite articles and talks about Beam.
```

Comment:
Replace "for"->"to view" to finish off this list's parallel structure.

Reply:
Thanks for the pointer; done!